chapter-10-workload-identities/dbclient/dbclient.yaml

```diff
@@ -23,17 +23,27 @@ spec:
         dnsIdentity: dbclient
     spec:
       serviceAccountName: dbclient
+      securityContext:
+        runAsGroup: 10001
+        runAsUser: 10001
       containers:
       - name: dbclient
         image: dbclient:v1
+        securityContext:
+          {}
         volumeMounts:
           - name: svids
             mountPath: /svids
       - name: dbclient-spiffehelper
-        image: dbclient-spiffehelper:v1
+        image: ghcr.io/spiffe/spiffe-helper:0.8.0
```

Comment:
Use the technique that [SPIRE uses](https://github.com/spiffe/helm-charts-hardened/blob/main/examples/mysql-using-spire/mysqlclient-statefulset.yaml#L50).